test/kubemark/start-kubemark.sh

```diff
@@ -225,6 +225,18 @@ function start-hollow-nodes {
   wait-for-hollow-nodes-to-run-or-timeout
 }
 
+# Annotates the node objects in the kubemark cluster to make their size
+# similar to regular nodes.
+function resize-node-objects {
+  # TODO: If KUBEMARK_NODE_OBJECT_SIZE_BYTES not set then do nothing
+  annotation_size_bytes=${KUBEMARK_NODE_OBJECT_SIZE_BYTES:-15000}
+
+  echo "Annotating node objects with ${annotation_size_bytes} byte label"
```

Comment:
I can live with something like that as a temporary thing, but medium term I would like to take advantage of this:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/imagepreload/imagepreload.go

[It's better reflecting the reality...]

So if you want to proceed with sth like that, please add an explicit TODO (and a bug) to remove it.

Reply:
The problem with imagepreload is that it won't work for kubemark, at least in the current form :( 
The thing is that it invovkes docker pull inside a container, which obviously won't work for kubemark.

But, I agree with you in general, the intention is to make this a temporary few-line workaround until we have a better fully fledged solution.

BTW, it's a WIP PR, not ready for review :)

Reply:
> The problem with imagepreload is that it won't work for kubemark, at least in the current form :(

I know - but it's possible to change the shim we use for docker (or CRI, not sure if we already migrated or not), so actually take images from the real node.
It requires work but it's doable (that's why I said, that I can live with it as a temporary thing).